server/src/internalClusterTest/java/org/opensearch/search/aggregations/ConcurrentSearchCardinalityWithRequestBreakerIT.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.search.aggregations;
+
+import org.opensearch.ExceptionsHelper;
+import org.opensearch.OpenSearchException;
+import org.opensearch.action.index.IndexRequestBuilder;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.util.FeatureFlags;
+import org.opensearch.core.common.breaker.CircuitBreakingException;
+import org.opensearch.indices.breaker.HierarchyCircuitBreakerService;
+import org.opensearch.test.OpenSearchIntegTestCase;
+
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import static org.opensearch.search.aggregations.AggregationBuilders.cardinality;
+import static org.opensearch.search.aggregations.AggregationBuilders.terms;
+
+/*
+* Creating a separate class for Concurrent Search use-case as parameterization doesn't work with
+* Circuit Breaker causing flaky tests. ref: https://github.com/opensearch-project/OpenSearch/issues/10154
+* */
+@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST)
+public class ConcurrentSearchCardinalityWithRequestBreakerIT extends OpenSearchIntegTestCase {
```

Comment:
Should this just extend `CardinalityWithRequestBreakerIT` directly and then all we need to do is override the `featureFlagSettings` method instead of copying any code?

Reply:
makes sense, i will make that change